manifests/l/LLVM/ClangFormat/19.1.7/LLVM.ClangFormat.yaml

```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: LLVM.ClangFormat
+PackageVersion: 19.1.7
+PackageLocale: en-US
+Publisher: LLVM
+PackageName: ClangFormat
+License: MIT
+ShortDescription: A tool to format C/C++/Java/JavaScript/JSON/Objective-C/Protobuf/C# code.
+Installers:
+- Architecture: x64
+  InstallerType: zip
+  InstallerUrl: https://github.com/PolarGoose/clang-format-for-Windows/releases/download/19.1.7/clang-format.zip
+  InstallerSha256: ff715d6995d74269a194a4f92adb4496a6e896110f2519210810941ed7d4a35b
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: ./clang-format.exe
+ManifestType: singleton
```

Comment:
Please be aware that the `singleton` manifest format has been depreciated, and consider using the [multi-file format](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0).
- https://github.com/microsoft/winget-pkgs/issues/181590